testdata/package/example-1.0.0/index.json

```diff
@@ -2,6 +2,7 @@
   "name": "example",
   "title": "Example Integration",
   "version": "1.0.0",
+  "readme": "/package/example-1.0.0/docs/README.md",
```

Comment:
@neptunian I made a change to the PR that this is now the full path like for all the other assets. Let me know what works best on your end.

Reply:
ah, i missed this previously.  this is preferable.